associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl

```diff
@@ -91,41 +99,66 @@
     </choose>
   </macro>
   <macro name="translator">
-    <text value="Traducao "/>
+    <text value="Tradução "/>
     <names variable="translator" delimiter="; ">
-      <name delimiter="; " sort-separator=" " delimiter-precedes-last="always">
-        <name-part name="given" text-case="capitalize-first"/>
-        <name-part name="family" text-case="capitalize-first"/>
-      </name>
+      <name delimiter="; " sort-separator=" " delimiter-precedes-last="always"></name>
+      <et-al font-style="italic"/>
     </names>
   </macro>
-  <macro name="author">
-    <names variable="author">
-      <name name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter="; " delimiter-precedes-last="always">
-        <name-part name="family" text-case="uppercase"/>
-        <name-part name="given" text-case="capitalize-first"/>
-      </name>
-      <label form="short" prefix=" (" suffix=".)" text-case="uppercase"/>
-      <substitute>
-        <names variable="editor"/>
-        <names variable="translator"/>
-        <text macro="title"/>
-      </substitute>
+  <macro name="composer">
+    <text value="Compositor: "/>
+    <names variable="composer" delimiter="; ">
+      <name delimiter="; " sort-separator=" " delimiter-precedes-last="always"></name>
+      <et-al font-style="italic"/>
+    </names>
+  </macro>
+  <macro name="recipient">
+    <text value="Destinatário: "/>
+    <names variable="recipient" delimiter="; ">
+      <name delimiter=" e " sort-separator=" " delimiter-precedes-last="always"></name>
```

Comment:
what's the goal of using "e" here? Wouldn't `and="text"` be better? With " e" as you have it, you'd e.g. get
> Leticia Strehl e Luísia Alves e Michelangelo Viana

whereas I think you probably want

> Leticia Strehl, Luísia Alves e Michelangelo Viana

It is probably rare for letters to have more than two recipients, but might as well get this right. If I'm right and you want the second option above then `<name and="text"/>` would be all you need. It's a bit odd that all the names have different formats, though? E.g. for composers you have 

> Leticia Strehl; Luísia Alves; Michelangelo Viana

Also, as you're currently handling them, the labels like "Compositor: " wouldn't work for multiple composers (which would be "Compositores: " I believe?)
That's what labels are for, along the lines of:
```
<names variable="composer" delimiter="; ">
  <label variabel="composer" text-case="capitalize-first" suffix=": "/>
  <name delimiter="; " sort-separator=" " delimiter-precedes-last="always"></name>
  <et-al font-style="italic"/>
</names>
```

Reply:
You are right! Thanks! But we will have to make these corrections at the next update (estimated for next July).

Reply:
@adam3smith can we merge this style for now?